string a
string b
---
float32 data0
float32 data1
float32 data2
